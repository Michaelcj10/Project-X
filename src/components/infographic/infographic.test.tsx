import * as React from "react";
import { render } from "@testing-library/react";
import Infographic from "./infographic";
import { ParallaxProvider } from "react-scroll-parallax";

describe("<Infographic /> spec", () => {
  it("renders the component", () => {
    const container = render(
      <ParallaxProvider>
        <Infographic />
      </ParallaxProvider>
    );
    expect(container).toMatchSnapshot();
  });
});
