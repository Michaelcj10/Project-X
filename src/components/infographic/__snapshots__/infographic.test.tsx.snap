// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Infographic /> spec renders the component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
        <div
            class="sc-AykKD crSbDL"
        >
            <div
                class="infographic-block container"
            >
                <div
                    class="row"
                >
                    <div
                        class="twelve columns centered"
                    >
                        <figure
                            class="parallax-outer custom-class"
                        >
                            <div
                                class="parallax-inner"
                            >
                                <h2>
                                    Puca Labs
                                </h2>
                                <img />
                            </div>
                        </figure>
                        <p>
                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed augue quam, egestas vitae elementum eu, blandit vitae felis. Maecenas lobortis vehicula risus sit amet elementum.
                        </p>
                    </div>
                </div>
                <figure
                    class="parallax-outer custom-class"
                >
                    <div
                        class="parallax-inner"
                    >
                        <div
                            class="sc-AykKC hhMQei"
                        >
                            <div
                                class="row"
                            >
                                <div
                                    class="one-half column centered padded"
                                >
                                    <div
                                        class="img-block greyed-on-touch"
                                    >
                                        <div>
                                            <img
                                                alt="Mobile"
                                            />
                                        </div>
                                    </div>
                                </div>
                                <div
                                    class="one-half column centered padded"
                                >
                                    <div>
                                        <h2>
                                            Creative designs
                                        </h2>
                                    </div>
                                    <div>
                                        <p>
                                            We make high quality , mobile responsive and SEO compliant websites. If you want to get full exposure for your business, or you just want a personal site to show off your holiday snaps, or wedding photos, we've got you covered!
                                        </p>
                                        <div
                                            class="dn-break"
                                        />
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </figure>
                <figure
                    class="parallax-outer custom-class"
                >
                    <div
                        class="parallax-inner"
                    >
                        <div
                            class="sc-AykKC hhMQei"
                        >
                            <div
                                class="row"
                            >
                                <div
                                    class="one-half column centered padded"
                                >
                                    <div>
                                        <h2>
                                            Mobile responsive
                                        </h2>
                                    </div>
                                    <div>
                                        <p>
                                            We make high quality , mobile responsive and SEO compliant websites. If you want to get full exposure for your business, or you just want a personal site to show off your holiday snaps, or wedding photos, we've got you covered!
                                        </p>
                                        <div
                                            class="dn-break"
                                        />
                                    </div>
                                </div>
                                <div
                                    class="one-half column centered padded"
                                >
                                    <div
                                        class="img-block greyed-on-touch"
                                    >
                                        <div>
                                            <img
                                                alt="Mobile"
                                            />
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </figure>
                <figure
                    class="parallax-outer custom-class"
                >
                    <div
                        class="parallax-inner"
                    >
                        <div
                            class="sc-AykKC hhMQei"
                        >
                            <div
                                class="row"
                            >
                                <div
                                    class="one-half column centered padded"
                                >
                                    <div
                                        class="img-block greyed-on-touch"
                                    >
                                        <div>
                                            <img
                                                alt="Mobile"
                                            />
                                        </div>
                                    </div>
                                </div>
                                <div
                                    class="one-half column centered padded"
                                >
                                    <div>
                                        <h2>
                                            Social integration
                                        </h2>
                                    </div>
                                    <div>
                                        <p>
                                            We make high quality , mobile responsive and SEO compliant websites. If you want to get full exposure for your business, or you just want a personal site to show off your holiday snaps, or wedding photos, we've got you covered!
                                        </p>
                                        <div
                                            class="dn-break"
                                        />
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </figure>
            </div>
        </div>
    </div>
</body>,
  "container": <div>
    <div
        class="sc-AykKD crSbDL"
    >
        <div
            class="infographic-block container"
        >
            <div
                class="row"
            >
                <div
                    class="twelve columns centered"
                >
                    <figure
                        class="parallax-outer custom-class"
                    >
                        <div
                            class="parallax-inner"
                        >
                            <h2>
                                Puca Labs
                            </h2>
                            <img />
                        </div>
                    </figure>
                    <p>
                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed augue quam, egestas vitae elementum eu, blandit vitae felis. Maecenas lobortis vehicula risus sit amet elementum.
                    </p>
                </div>
            </div>
            <figure
                class="parallax-outer custom-class"
            >
                <div
                    class="parallax-inner"
                >
                    <div
                        class="sc-AykKC hhMQei"
                    >
                        <div
                            class="row"
                        >
                            <div
                                class="one-half column centered padded"
                            >
                                <div
                                    class="img-block greyed-on-touch"
                                >
                                    <div>
                                        <img
                                            alt="Mobile"
                                        />
                                    </div>
                                </div>
                            </div>
                            <div
                                class="one-half column centered padded"
                            >
                                <div>
                                    <h2>
                                        Creative designs
                                    </h2>
                                </div>
                                <div>
                                    <p>
                                        We make high quality , mobile responsive and SEO compliant websites. If you want to get full exposure for your business, or you just want a personal site to show off your holiday snaps, or wedding photos, we've got you covered!
                                    </p>
                                    <div
                                        class="dn-break"
                                    />
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </figure>
            <figure
                class="parallax-outer custom-class"
            >
                <div
                    class="parallax-inner"
                >
                    <div
                        class="sc-AykKC hhMQei"
                    >
                        <div
                            class="row"
                        >
                            <div
                                class="one-half column centered padded"
                            >
                                <div>
                                    <h2>
                                        Mobile responsive
                                    </h2>
                                </div>
                                <div>
                                    <p>
                                        We make high quality , mobile responsive and SEO compliant websites. If you want to get full exposure for your business, or you just want a personal site to show off your holiday snaps, or wedding photos, we've got you covered!
                                    </p>
                                    <div
                                        class="dn-break"
                                    />
                                </div>
                            </div>
                            <div
                                class="one-half column centered padded"
                            >
                                <div
                                    class="img-block greyed-on-touch"
                                >
                                    <div>
                                        <img
                                            alt="Mobile"
                                        />
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </figure>
            <figure
                class="parallax-outer custom-class"
            >
                <div
                    class="parallax-inner"
                >
                    <div
                        class="sc-AykKC hhMQei"
                    >
                        <div
                            class="row"
                        >
                            <div
                                class="one-half column centered padded"
                            >
                                <div
                                    class="img-block greyed-on-touch"
                                >
                                    <div>
                                        <img
                                            alt="Mobile"
                                        />
                                    </div>
                                </div>
                            </div>
                            <div
                                class="one-half column centered padded"
                            >
                                <div>
                                    <h2>
                                        Social integration
                                    </h2>
                                </div>
                                <div>
                                    <p>
                                        We make high quality , mobile responsive and SEO compliant websites. If you want to get full exposure for your business, or you just want a personal site to show off your holiday snaps, or wedding photos, we've got you covered!
                                    </p>
                                    <div
                                        class="dn-break"
                                    />
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </figure>
        </div>
    </div>
</div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
